sample/src/main/kotlin/com/chuckerteam/chucker/sample/MainActivity.kt

```diff
@@ -38,6 +38,13 @@ class MainActivity : AppCompatActivity() {
             launchChuckerDirectly.visibility = if (Chucker.isOp) View.VISIBLE else View.GONE
             launchChuckerDirectly.setOnClickListener { launchChuckerDirectly() }
 
+            exportToFile?.visibility = if (Chucker.isOp) View.VISIBLE else View.GONE
+            exportToFile?.setOnClickListener {
+                Thread {
+                    Chucker.writeTransactions(this@MainActivity, 1, null)
+                }.start()
```

Comment:
Please don't start a thread explicity here. We use coroutines to do any async work.

Reply:
Have updated to use coroutines, but have had to add coroutines and activity-ktx dependencies to the sample gradle project as they weren't there before

Reply:
Yeah I realized this is a side effect 🤔 I'll be fine with it. I'm unsure what's the other maintainers take on this (specifically @MiSikora )